recipes/octo-encryption-cpp/all/conanfile.py

```diff
@@ -0,0 +1,97 @@
+from conans import ConanFile
+from conan.tools.cmake import CMake, CMakeToolchain, cmake_layout
+from conan.tools.files import get
+from conan.tools.build import check_min_cppstd
+from conans.errors import ConanInvalidConfiguration
+from conan.tools.scm import Version
+
+
+class OctoEncryptionCPPConan(ConanFile):
+    name = "octo-encryption-cpp"
+    license = "MIT"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/ofiriluz/octo-encryption-cpp"
+    description = "Octo encryption library"
+    topics = ("cryptography", "cpp")
+    settings = "os", "compiler", "build_type", "arch"
+
+    @property
+    def _source_subfolder(self):
+        return "source"
+
+    @property
+    def _build_subfolder(self):
+        return "build"
+
+    @property
+    def _compilers_minimum_version(self):
+        return {
+            "gcc": "8",
+            "clang": "9",
+            "apple-clang": "11",
+            "Visual Studio": "16",
+        }
+
+    def generate(self):
+        tc = CMakeToolchain(self)
+        tc.generate()  
+
+    def layout(self):
+        cmake_layout(self, src_folder=self._source_subfolder)
```

Comment:
```suggestion
        cmake_layout(self, src_folder="src")
```
Because on the project, source files are under src/ folder

Reply:
ya, i revamped the conanfile post your suggestion, this is already changed :)